src/js/lib.js

```diff
@@ -1,5 +1,6 @@
 export const createRootElement = () => {
-  const injectionElement = document.querySelector('.pr-toolbar')
+  const injectionElement = document.querySelector('#files_bucket')
+  const nextSibiling = document.querySelector('.diff-view')
```

Comment:
Typo, awesome work anyway 👏 

```suggestion
  const nextSibling = document.querySelector('.diff-view')
```